starter_code/models/movie.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const movieSchema = new Schema({
+    title: String,
+    director: String,
+    stars: [],
```

Comment:
You can also put it like this to make it a bit more clear that it is an array of strings only:
stars: [String]